api/apps/api/src/modules/authentication/authentication.service.ts

```diff
@@ -126,16 +136,15 @@ export class AuthenticationService {
     });
     const validationToken = v4();
     await this.apiEventsService.create({
```

Comment:
Let's discuss it how we can avoid messing with ApiEvents, as those purpose was most likely something else. We could even grab dedicated module handling account activation if necessary.

Reply:
yeah, that sounds good, I'll do that :+1: thanks Andrea :bow: